app/views/state_file/questions/ny_county/edit.html.erb

```diff
@@ -1,11 +1,52 @@
-<% title = t(".title", filing_year: Rails.configuration.statefile_current_tax_year  ) %>
-<% content_for :page_title, title %>
+<%
+title = t(".title", filing_year: @filing_year )
+content_for :page_title, title
+%>
+
+<% content_for :script do %>
+  <script>
+      window.addEventListener("load", () => {
+          const nycCounties = [
+              "Bronx",
+              "Kings (Brooklyn)",
+              "Manhattan (see New York)",
+              "New York (Manhattan)",
+              "Queens",
+              "Richmond (Staten Isand)"
+          ]
+          const nycResidency = <%= @nyc_residency == "full_year" %>
+
+          function displayWarnings(){
```

Comment:
so....i have not written javascript in a long time but should there be some semicolons in here...?